### PR TITLE
Calculate min utxo lovelace for coin only or with native assets

### PR DESCRIPTION
**Pull request by lilpaul**
Tested transaction submission on testnet with 1 less lovelace than the calculation, didnt work, and then tested with the actual amount calculated and it did work:

https://testnet.cardanoscan.io/transaction/d7914a9205e3d8774aed90ff39e22fe2fbb3d8f8874d7c239b4678767195f0bd

Calculation from keefie as per discord chat

### CODE REVIEW

**Comment by nothingalike** [comment]
@lilpaul is this for calculating how much ada you should keep in your wallet to accommodate your native assets?

**Comment by lilpaul** [comment]
> @lilpaul is this for calculating how much ada you should keep in your wallet to accommodate your native assets?

It's to calculate the minimum ada a transaction output requires.

File: CardanoSharp.Wallet.Test/TransactionOutputTests.cs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                .AddToken(getGenesisPolicyId(), "00010203".HexToByteArray(), 60)
+                .AddToken(getGenesisPolicyId(), "00010204".HexToByteArray(), 240);
+
+            var tokenBundle2 = TokenBundleBuilder.Create
+                .AddToken(getGenesisPolicyId(), "00010205".HexToByteArray(), 60)
+                .AddToken(getGenesisPolicyId(), "00010206".HexToByteArray(), 120)
+                .AddToken(getGenesisPolicyId(), "00010207".HexToByteArray(), 180)
+                .AddToken(getGenesisPolicyId(), "00010208".HexToByteArray(), 240);
+
+            var transactionBody = TransactionBodyBuilder.Create
```
Comment (safestak-keith) [inline CardanoSharp.Wallet.Test/TransactionOutputTests.cs:87]: We should be able to simplify this test to only involve TransactionOutput objects and asserting calculated values against the expected values?

Comment (lilpaul) [reply CardanoSharp.Wallet.Test/TransactionOutputTests.cs:87]: yeah good point, will put some time into simplifying and improving the tests

Comment (lilpaul) [reply CardanoSharp.Wallet.Test/TransactionOutputTests.cs]: trimmed down to just testing the bundles and the coin only calculation

**Pull request merged**